trade-finance-manager-api/src/v1/api.js

```diff
@@ -632,7 +632,7 @@ const updateGefFacility = async (facilityId, facilityUpdate) => {
       method: 'put',
       url: `${DTFS_CENTRAL_API_URL}/v1/portal/gef/facilities/${facilityId}`,
       headers: headers.central,
-      data: facilityUpdate,
+      data: { facilityUpdate, auditDetails }, // TODO: DTFS2-6966
```

Comment:
If this comment is still needed could you add a bit of detail please :)

Reply:
removed